.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: python
+arch:
+  - AMD64
```

Comment:
Can you use lowercase here too? E.g. https://docs.travis-ci.com/user/multi-cpu-architectures/#multi-cpu-availaibility

Reply:
yeah Done with changes, and pushed the code and merged into single commit.